internal/fields/mappings.go

```diff
@@ -152,6 +153,7 @@ func createValidatorForMappingsAndPackageRoot(fieldsParentDir string, finder pac
 		return v, nil
 	}
 
+	// TODO: Should we remove this code to load external and local fields?
 	fieldsDir := filepath.Join(fieldsParentDir, "fields")
```

Comment:
This code to load the external and local fields maybe can be removed. That would mean that just the schema set (optionally) via options is taken into account.

WDYT?

Reply:
Yes, sounds good.